src/vacuum-card.js

```diff
@@ -266,9 +266,14 @@ class VacuumCard extends LitElement {
         return html``;
       }
 
-      const value = entity_id
+      const [attributePrefix, attributeSuffix] = attribute.split('.');
```

Comment:
I'm not sure this is a good solution. I believe we should cover 3, 4, and more levels, so we should use some kind of recursive accessing.

Reply:
I see what you mean, but do you think it would be reasonable to add this in case there's a real use case for such a nesting? For instance, if there's a new community firmware which would expose data in such a way. I'm just thinking that 2 most popular options is stock and valetudo this approach should be reasonable for the time being.

Reply:
@krasnoukhov I don't know such a case. However, I didn't know about your case in the first place. I thought my solution covers most of the cases as well 🙂

Reply:
Yeah that totally makes sense. I'm quite sure that [valetudo](https://github.com/Hypfer/Valetudo/blob/61902edd8b185d116dd27a02827866e3defd01b1/lib/MqttClient.js#L369) (and [valetudo RE](https://github.com/rand256/valetudo/blob/2b8e6dfb9cefc380caa3c5951525e99537ef0ed0/lib/MqttClient.js#L335) which is similar) are the only options aside from stock firmware we need to deal with atm so I'd prefer to not overcomplicate this. Do you have a strong concern?

Reply:
Yep, I do. I believe this has to be universal. You can install [lodash.get](https://www.npmjs.com/package/lodash.get) to keep this simple.